gradle.properties

```diff
@@ -1,8 +1,8 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.2.3-SNAPSHOT
+version=1.3.0-SNAPSHOT
 #dependency versions
-ballerinaLangVersion=2201.0.1
+ballerinaLangVersion=2201.1.0-rc1.1
```

Comment:
```suggestion
ballerinaLangVersion=2201.1.0-rc1.2
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/797/commits/d392fe81bc4e919e2964dc387581557991582525